purity/gc/gc.go

```diff
@@ -239,6 +239,7 @@ func (c *collector) MarkReferencedBlobs() error {
 	wg.Add(c.parallelism + 1)
 	step := len(c.actionResults) / c.parallelism
 	for i := 0; i < (c.parallelism + 1); i++ {
+		i := i
```

Comment:
Is this useful? `i` is not used in the goroutine, so this should not be required